### DISCUSSION TITLE
How to reset callbacks when calling "load_from_checkpoint"

### DISCUSSION THREAD

**Discussion by jannisborn** [Lightning-AI/pytorch-lightning#13013] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
Say I'm loading pretrained model using `load_from_checkpoint(ckpt_path)`. If I used callbacks during the pretraining, the callback values will be restored (e.g., the minimal loss). 
This is very useful if one continues to train on **the same data** but it is prohibitive if the model is finetuned on another dataset. 
I'm looking for an option to _reset_ all callback values. Is this supported?

I know that I can overwrite hyperparameters (the [docs](https://pytorch-lightning.readthedocs.io/en/stable/common/lightning_module.html?highlight=load_from_checkpoint#load-from-checkpoint) are pretty verbose about it) but that's not what I want. 
Thanks for any advice!

**Comment by rohitgr7** [comment]
`load_from_checkpoint` just loads the model weights. It doesn't load the callback states.
